docs/source/notebooks/table_of_contents_examples.js

```diff
@@ -53,5 +53,5 @@ Gallery.contents = {
     "normalizing_flows_overview": "Variational Inference",
     "gaussian-mixture-model-advi": "Variational Inference",
     "GLM-hierarchical-advi-minibatch": "Variational Inference",
-    "ODE_parameter_estimation": "Inference in ODE models"
```

Comment:
If we are unlinking this NB (should we actually do that?) we should probably also remove the NB.